### PR TITLE
Move graphql code to separate workspace

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by tbergq** [comment]
Yeah, I was even considering example-relay-graphql. 

Maybe we should have a better name for it before merging.

**Comment by mrtnzlml** [comment]
👍 for `example-relay-graphql` - it's a best practice to use a common prefix for related projects in our setup.

File: src/graphql/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@adeira/graphql",
+  "private": false,
```
Comment (mrtnzlml) [inline src/graphql/package.json]: Should be private, no? It's not being published to NPM.

File: src/graphql/.npmignore [hunk 2]
```diff
@@ -0,0 +1 @@
+__tests__
```
Comment (mrtnzlml) [inline src/graphql/.npmignore]: This should not be needed for private packages.

**Comment by tbergq** [comment]
If it is not on npm, how will it be deployed as a part of the relay-example project?

**Comment by tbergq** [comment]
Closing, we will keep it within the `example-relay` for now

**Pull request closed without merging**